### PR TITLE
addAndBuy

### PR DESCRIPTION
**Pull request by germange**

### CODE REVIEW

File: pageObject/homePage.js [hunk 1]
```diff
@@ -3,12 +3,15 @@ let BasePage = require('../base/bagePage');
...
 let TextBox = require('../elements/textBox');
 let BaseElement = require('../base/baseElement');
+let DropDown = require('../elements/dropdown');
 
 let loginbuttonLocator = by.css('a.login');
 let searchTopInputLocator = by.css('#search_query_top');
 let submitSearchTopButtonLocator = by.css('.button-search');
 let searchResultsTextBoxLocator = by.css('.heading-counter');
 let firstSearchResultButton = by.css('.product_list.row > li:nth-child(1) h5 > a');
+let womenMenuDropdownLocator = by.xpath('//*[@id="block_top_menu"]/ul/li[1]');
```
Comment (nbabii) [inline pageObject/homePage.js]: optimize locator

File: pageObject/homePage.js [hunk 2]
```diff
@@ -3,12 +3,15 @@ let BasePage = require('../base/bagePage');
...
+let womenMenuTShirtButtonLocator = by.xpath('//*[@id="block_top_menu"]//li[1]//li[1]//li[1]/a');
```
Comment (nbabii) [inline pageObject/homePage.js]: optimize locator

File: pageObject/summaryPage.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    }
+
+    async quantityPlusClick() {
+        await allure.createStep(`Click Quantity Plus Button`, async () => {
+            await this.getQuantityPlusButton().multiclick(1);
+        })();
+    }
+
+    async quantityInputValueWaitForUpdate() {
+        await browser.sleep(2000);
```
Comment (nbabii) [inline pageObject/summaryPage.js]: No `browser.sleep` only browser.wait

File: pageObject/tShirtPage.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+let TextBox = require('../elements/textBox');
+let BaseElement = require('../base/baseElement');
+let CheckBox = require('../elements/checkbox');
+
+
+let tShirtHeaderLocator = by.css('.cat-name');
+let listViewCheckBoxLocator = by.css('#list');
+let addToCartButtonLocator = by.css('[title="Add to cart"]');
+let productAddedTextBoxLocator = by.css('.icon-ok');
+let proceedToCheckoutButtonLocator = by.xpath('//*[@id="layer_cart"]/div[1]/div[2]/div[4]/a/span');
```
Comment (nbabii) [inline pageObject/tShirtPage.js]: not good locator

**Review by nbabii — CHANGES_REQUESTED** [review]

File: pageObject/homePage.js (hunk 2 above)
Comment (germange) [reply pageObject/homePage.js]: do not have an ideas how to fix it

Comment (nbabii) [reply pageObject/homePage.js]: @germange check `.sfHover a[title="T-shirts"]`

File: pageObject/homePage.js (hunk 1 above)
Comment (nbabii) [reply pageObject/homePage.js]: Please check also this locator

**Review by nbabii — CHANGES_REQUESTED** [review]

**Pull request merged**